src/web/components/Img.tsx

```diff
@@ -48,6 +48,17 @@ const buildSourcesString = (srcSets: SrcSetItem[]): string => {
     return srcSets.map((srcSet) => `${srcSet.src} ${srcSet.width}w`).join(',');
 };
 
+/**
+ *       mobile: 320
+ *       mobileMedium: 375
+ *       mobileLandscape: 480
```

Comment:
how accurate is that, nowadays?

Reply:
Probably not very, the name is certainly confusing for me. We design to it though so it would be hard to remove.

Reply:
The names are somewhat arbitrary. We need to specify breakpoints _somewhere_ - we could probably name them 1-7 if we wanted to 🤷‍♂️.